### ISSUE TITLE
Failed validation: #/definitions/QuoteSummaryResult

### ISSUE THREAD

**Issue by EcceGratum** [gadicc/node-yahoo-finance2#639]
# Validation Error

## Symbol(s) that it happened for
- HISU-U.TO

## Error Message

```
The following result did not validate with schema: #/definitions/QuoteSummaryResult
[
  {
    instancePath: '/topHoldings',
    schemaPath: '#/required',
    keyword: 'required',
    params: { missingProperty: 'stockPosition' },
    message: "must have required property 'stockPosition'",
    data: {
      maxAge: 1,
      holdings: [],
      equityHoldings: {
        priceToEarnings: 0,
        priceToBook: 0,
        priceToSales: 0,
        priceToCashflow: 0
      },
      bondHoldings: {},
      bondRatings: [],
      sectorWeightings: []
    }
  },
  {
    instancePath: '/topHoldings',
    schemaPath: '#/required',
    keyword: 'required',
    params: { missingProperty: 'bondPosition' },
    message: "must have required property 'bondPosition'",
    data: {
      maxAge: 1,
      holdings: [],
      equityHoldings: {
        priceToEarnings: 0,
        priceToBook: 0,
        priceToSales: 0,
        priceToCashflow: 0
      },
      bondHoldings: {},
      bondRatings: [],
      sectorWeightings: []
    }
  }
]
```

## Environment
Browser or Node: 
Node version (if applicable): 
Npm version: 
Browser verion (if applicable):
Library version (e.g. `1.10.1`): 

## Additional Context
https://github.com/ghostfolio/ghostfolio/issues/1966

**Comment by Xeli** [comment]
I'm experiencing the same issue with a bunch of ETFs.

- 0P00005ZKV.F
- 0P0001IJJY.F
- 0P0000WKQV.F
getting the same error as the original posted had. For reference this is the page in yahoo-finance: https://finance.yahoo.com/quote/0P0000WKQV.F/holdings?p=0P0000WKQV.F

**Comment by bmacedo** [comment]
Same failure happens for the ETF CSH2.L

https://finance.yahoo.com/quote/CSH2.L/holdings?p=CSH2.L

**Comment by gadicc** [comment]
Thanks, @EcceGratum, @Xeli, @bmacedo for reporting and for your patience.

This is now fixed and released, as per the above.

I've also added a section in the validation docs about how to help contribute these kinds of fixes, if anyone is interested :)  https://github.com/gadicc/node-yahoo-finance2/blob/devel/docs/validation.md#help-fix

The changes are usually very simple, e.g.:

Error: `message: "must have required property 'stockPosition'"` (and bondPosition)

Modify `src/modules/quoteSummary-iface.ts`:

```diff
export interface TopHoldings {
  [key: string]: any;
  maxAge: number;
-  stockPosition: number;
-  bondPosition: number;
+  stockPosition?: number;
+  bondPosition?: number;
  holdings: TopHoldingsHolding[];
  equityHoldings: TopHoldingsEquityHoldings;
  bondHoldings: object;
```

and of course, a little more time for testing, etc.  I'm just so overloaded at the moment so some help would be great :)
